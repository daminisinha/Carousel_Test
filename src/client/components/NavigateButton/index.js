import React, { Component } from 'react';

class NavigateButton extends Component {

    render() {
        let {buttonText,onClickHandler} = this.props;
        return (
                <button name={buttonText} onClick={onClickHandler} >{buttonText}</button>
        );
    }
}

export default NavigateButton;