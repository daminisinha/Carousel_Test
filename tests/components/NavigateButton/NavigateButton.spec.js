import React from 'react';
import NavigateButton from '../../../src/client/components/NavigateButton'
import {shallow} from 'enzyme';

test('Navigate button renders',()=>{
    const component = shallow(<NavigateButton/>); 
    expect(component).toMatchSnapshot();
})
