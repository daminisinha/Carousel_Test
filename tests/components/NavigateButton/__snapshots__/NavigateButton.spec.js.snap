// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigate button renders 1`] = `<button />`;
